packages/components/src/Disclosure/Disclosure.css

```diff
@@ -24,15 +28,18 @@
 }
 
 .summary:hover,
-.summary:focus {
-  background: var(--color-surface--hover);
+.summary:focus-visible {
+  background-color: var(--color-surface--hover);
 }
 
 .summary:focus,
 .summary:active {
-  box-shadow: var(--shadow-focus);
   border: none;
-  outline: none;
+  outline: transparent;
```

Comment:
High-contrast mode ignores box-shadows, and forces an outline color. `none` blocks high-contrast mode from working as expected, transparent is compatible, with no visual impact to non-high-contrast mode users.